code/modules/projectiles/guns/ballistic.dm

```diff
@@ -384,12 +388,24 @@
 #undef BRAINS_BLOWN_THROW_SPEED
 #undef BRAINS_BLOWN_THROW_RANGE
 
-
-
-/obj/item/gun/ballistic/proc/sawoff(mob/user)
+GLOBAL_LIST_INIT(gun_saw_types, typecacheof(list(
+	/obj/item/circular_saw,
+	/obj/item/gun/energy/plasmacutter,
+	/obj/item/melee/transforming/energy,
+	/obj/item/twohanded/required/chainsaw,
+	/obj/item/nullrod/claymore/chainsaw_sword,
+	/obj/item/nullrod/chainsaw,
```

Comment:
once again, the chainsaw hand null rod form proves superior to the INFERIOR dark blessing null rod form (it was already superior before this PR due to having a better butchering speed)

anyway, this (chainsaw hand null rods being able to saw off shotguns) makes sense and I approve of it

Reply:
with dark blessing you can cause a validhunting storm so id argue its better